README.md

```diff
@@ -267,36 +500,87 @@ const BookList = ({ genre }) => {
   );
 };
 
-BookList.someServerSideFunc = async ({ store: { dispatch, getState } }) => {
-  const fetchye = makeServerFetchye({
-    // Redux store
-    store: { dispatch, getState },
-    fetchClient: fetch,
-    // Selector to wherever fetchye reducer exists in Redux
-    cacheSelector: (state) => state,
-  });
+BookList.holocron = {
+  loadModuleData: async ({ store: { dispatch, getState }, fetchClient }) => {
+    if (global.BROWSER) {
+      return;
+    }
+    const fetchye = makeServerFetchye({
+      // Redux store
+      store: { dispatch, getState },
+      // Use ImmutableCache as One App uses ImmutableJS
+      cache: ImmutableCache({
+        // Selector to wherever fetchye reducer exists in Redux
+        cacheSelector: (state) => state.getIn(['modules', 'my-module-root', 'fetchye']),
+      }),
+      fetchClient,
+    });
+
+    // async/await fetchye has same arguments as useFetchye
+    // dispatches events into the server side Redux store
+    await fetchye('http://example.com/api/books/');
+  },
+};
 
-  // async/await fetchye has same arguments as useFetchye
-  const { data, error } = await fetchye(`http://example.com/api/books/?genre=${genre}`, {
-    headers: {
-      'X-Some-Tracer-Id': 1234,
-    },
+export default BookList;
+```
+
+#### Next.JS SSR
+
+```jsx
+import SimpleCache from 'fetchye/lib/cache/SimpleCache';
+import { useFetchye, makeServerFetchye } from 'fetchye';
+
+const cache = SimpleCache();
+
+// Codesandbox takes a second to get Next.JS started...
+export default function IndexPage({ initialBookList }) {
+  const { isLoading, data } = useFetchye('http://example.com/api/books/', {
+    initialData: initialBookList,
   });
 
-  // Server will run a fetchye API call and cache the result.
-  // Client will draw the API call result with matching arguments from the cache
-  // without firing an API call.
-};
+  if (isLoading) {
+    return (<p>Loading...</p>);
+  }
+
+  return (
+    <>
+      {/* Render data */}
+    </>
+  );
+}
+
+const fetchye = makeServerFetchye({
+  cache,
+  fetchClient: fetch,
+});
+
+export async function getServerSideProps() {
+  try {
+    // returns { data, error } payload for feeding initialData on client side
+    const res = await fetchye('http://example.com/api/books/');
+    return {
+      props: {
+        initialBookList: res,
+      },
+    };
+  } catch (error) {
+    console.error(error.message);
+    return {};
+  }
+}
 ```
 
+🏖️[Try this out on Codesandbox](https://codesandbox.io/s/americanexpressfetchye-fetchye-ssr-0ktx9)
+
 ## 🎛️ API
```

Comment:
What do you think about having something similar to this https://github.com/nellyk/holocron/tree/main/packages/holocron/docs/api

Reply:
especially with the table of contents

Reply:
Yes, it definitely could use a table of contents.

Reply:
Fixed in https://github.com/americanexpress/fetchye/commit/3abb96435114948636c3f657a0364b265568ff91